saleor/graphql/translations/mutations.py

```diff
@@ -24,6 +26,11 @@ def check_permissions(cls, context):
 
     @classmethod
     def perform_mutation(cls, _root, info, **data):
+        if "id" in data and not data["id"]:
+            raise ValidationError(
+                {"id": ValidationError("This field is required", code="required")}
+            )
+
```

Comment:
I am curious, `required=True` on the input field is not enough?

Reply:
It still allows you to pass `id="` which fails as the `instance` is None then.